etcd3/client.py

```diff
@@ -177,10 +179,14 @@ def __enter__(self):
 
     def __exit__(self, *args):
         self.close()
-        
+
     def __del__(self):
         self.close()
 
+    @property
+    def lock_prefix(self):
```

Comment:
can you remove this `@property` too please

Reply:
This was added to enable accessing the `_lock_prefix` private variable of `Etcd3Client`. Should I make that a public variable instead in that case?